samples/ra-vpn/saml-multi-region/enforcers-asaconf.yaml

```diff
@@ -0,0 +1,121 @@
+# This config is not region-specific, and can be applied to any of your clusters.
+# ASAConfigurations sensitive to region can be found in ./eu-region/ and ./us-region
+# respectively. 
+apiVersion: cnfw.cisco.com/v1
+kind: ASAConfiguration
+metadata:
+  name: ravpn-enforcer-config
+  namespace: sfcn-system
+  labels:
+    sfcn.cisco.com/service-role: "default"
+spec:
+  order: 1
+  description: "RA-VPN Configuration"
+  fileObjects:
+    - "ravpnprofile"
+    - "anyconnectlinux"
+    - "anyconnectwin"
+    - "anyconnectmac"
+  ipv4SubnetPools:
+    - "ravpnpool"
+  secrets:
+    - "sfcn-redis"
+    - "mypkcs"
+    - "samlpkcs"
+  cliLines: |
+    interface Management0/0
+      no management-only
+      nameif management
+      security-level 0
+      ip address dhcp
+    interface TenGigabitEthernet0/0
+      nameif outside
+      security-level 0
+      ip address dhcp
+    interface TenGigabitEthernet0/1
+      nameif inside
+      security-level 100
+      ip address dhcp
+    # Configure route to internet over outside interface
+    route outside 0 0 {{ index .nodeLabels "sfcn.cisco.com.interface.2/gateway-ipv4" }} 1
+    # configure route to vpc addresses over inside interface
+    route inside 10.37.0.0 255.255.0.0 {{ index .nodeLabels "sfcn.cisco.com.interface.3/gateway-ipv4" }} 2
+    
+    dns domain-lookup outside
+    dns server-group DefaultDNS
+    name-server 169.254.169.253
+
+    # Configure the IP address pool from where the clients will receive an IP address
+    ip local pool VPN_POOL {{.ipv4SubnetPools.ravpnpool.assignedRange}} mask 255.255.255.0
+    access-list Split_Tunnel_ACL extended permit ip 10.37.0.0 255.255.0.0 any4
+    vpn load-balancing
```

Comment:
Please confirm once that load-balancing entry is required for enforcers..

Reply:
> Please confirm once that load-balancing entry is required for enforcers..

cc @kkelkar

Reply:
The `external-database` and `priority` configuration are required to participate in load balancing and `redirect-fqdn enable` is needed so that the SAML assertions have the FQDN instead of IP